integration_tests/run_pyspark_from_build.sh

```diff
@@ -46,27 +46,48 @@ else
 
     # support alternate local jars NOT building from the source code
     if [ -d "$LOCAL_JAR_PATH" ]; then
-        AVRO_JARS=$(echo "$LOCAL_JAR_PATH"/spark-avro*.jar)
-        PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark_*.jar)
-        if [ -f $(echo $LOCAL_JAR_PATH/parquet-hadoop*.jar) ]; then
+        if [ -f "$LOCAL_JAR_PATH"/spark-avro*.jar ]; then
+            AVRO_JARS=$(echo "$LOCAL_JAR_PATH"/spark-avro*.jar)
```

Comment:
nit: these echos can be `$(readlink -f  path)` which would get of `..` for some of the paths we use.

Reply:
Done